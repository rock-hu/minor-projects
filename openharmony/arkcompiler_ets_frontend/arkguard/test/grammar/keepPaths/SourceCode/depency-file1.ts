/*
 * Copyright (c) 2024 Huawei Device Co., Ltd.
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 *     http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */

export class DependencyClass {
  depencyProp1: string = "";
  depencyProp2: number = 1;
  depencyMethod1() {
    const tempValue = 2;
    return this.depencyProp1 + this.depencyProp2;
  }
}

export function DependencyFunctionReduce(num1: number, num2: number) {
  return num1 - num2;
}


export interface DependencyInterface {
  depencyProp3: string;
  depencyProp4: number;
}