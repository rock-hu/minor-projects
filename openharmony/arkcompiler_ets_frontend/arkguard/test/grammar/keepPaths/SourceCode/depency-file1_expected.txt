export class DependencyClass {
    depencyProp1: string = "";
    depencyProp2: number = 1;
    depencyMethod1() {
        const c = 2;
        return this.depencyProp1 + this.depencyProp2;
    }
}
export function DependencyFunctionReduce(a: number, b: number) {
    return a - b;
}
export interface DependencyInterface {
    depencyProp3: string;
    depencyProp4: number;
}
